src/Illuminate/Routing/Pipeline.php

```diff
@@ -69,6 +75,11 @@ protected function getInitialSlice(Closure $destination)
      */
     protected function handleException($passable, Exception $e)
     {
+        if (self::$exceptionLevel >= 3) {
+            die($e->getMessage());
```

Comment:
https://github.com/laravel/framework/issues/14518#issuecomment-235865890
> It doesn't pose a security issue, and it's nothing really to do with Laravel since PHP itself provides this exception and it's message.

The idea is, getting the worst/ugly text message is better than get nothing.